src/v0/destinations/branch/utils.js

```diff
@@ -0,0 +1,24 @@
+const { getHashFromArray } = require('../../util');
+
+/**
+ * Retrieves the mapped event name from the given config.
+ *
+ * @param {object} message - The message object containing the event.
+ * @param {object} destination - The destination object containing the events mapping configuration.
+ * @returns {string} - The mapped event name, or undefined if not found.
+ */
+const getMappedEventNameFromConfig = (message, destination) => {
+  let eventName;
+  const { event } = message;
+  const { eventsMapping } = destination.Config;
+
+  // if event is mapped on dashboard, use the mapped event name
+  if (Array.isArray(eventsMapping) && eventsMapping.length > 0) {
+    const keyMap = getHashFromArray(eventsMapping, 'from', 'to');
```

Comment:
This is behaviour for majority of the destination where getHashFromArray is used.

Reply:
In general, we should not make any assumptions. Moreover, the proposed solution is more future-proof.

Say events ` a' and `A` are in the user's instrumentation.
If we treat them case-insensitively but the user doesn't want us to, then there is no way to handle it without changing our code (and neatly handling backward compatibility for other users).

However, if we treat them case-sensitively by default, but the user wants otherwise, they can quickly write a UT to address it for their use case.

-----------------
There was a destination that required case sensitivity, so we added a new default argument to the utility to handle it.

Reply:
Got it, Updated the code to address this